01week/rockPaperScissors.js

```diff
@@ -9,9 +9,27 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+  if (hand1 === hand2) {
```

Comment:
Remember to trim and convert your input strings to a uniform capitalization so your program doesn't do funny things with funny inputs.